packages/sdk/src/AllowLists/SimpleDenyList.ts

```diff
@@ -83,10 +84,12 @@ export class SimpleDenyList<
    *
    * @public
    * @static
-   * @type {Address}
+   * @type {Record<number, Address>}
    */
-  public static override base: Address = import.meta.env
-    .VITE_SIMPLE_DENYLIST_BASE;
+  public static override bases: Record<number, Address> = {
+    ...(SimpleDenyListBases as Record<number, Address>),
+    1337: import.meta.env.VITE_SIMPLE_DENYLIST_BASE,
```

Comment:
what is the practical significance of chain id `1337` now? Don't all test chains (anvil, hardhat) use `31337` at this point?